.github/workflows/lint.yml

```diff
@@ -44,6 +65,11 @@ jobs:
       contents: read
     runs-on: ubuntu-latest
     steps:
+      - uses: step-security/harden-runner@main
+        with:
+          allowed-endpoints:
+            agent.api.stepsecurity.io:443
+            github.com:443
```

Comment:
for example, the `actions/checkout` action both always should be able to access github.com, and also, is the only part of the action that needs to

Reply:
Yes, I understand what you are saying, and that would be ideal. I wrote about this in the comment above.

Reply:
**